java-agent/src/main/java/io/opentelemetry/auto/bootstrap/AgentBootstrap.java

```diff
@@ -55,6 +56,7 @@
  * </ul>
  */
 public class AgentBootstrap {
+  private static final Class<?> thisClass = MethodHandles.lookup().lookupClass();
```

Comment:
Why is this better than `AgentBootstrap.class`?

Reply:
Looks like it could be either: https://github.com/DataDog/dd-trace-java/pull/1561/files#r437482125

Reply:
I argue that `AgentBootstrap.class` is much easier to understand.

Reply:
done